wellindexcalculator.cpp

```diff
@@ -27,32 +27,39 @@
 
 namespace Reservoir {
     namespace WellIndexCalculation {
+        using namespace std;
+
         WellIndexCalculator::WellIndexCalculator(Grid::Grid *grid) {
             grid_ = grid;
         }
 
-        std::vector<IntersectedCell> WellIndexCalculator::ComputeWellBlocks(Vector3d heel, Vector3d toe, double wellbore_radius) {
-            heel_ = heel;
-            toe_ = toe;
+        vector<IntersectedCell>
+        WellIndexCalculator::ComputeWellBlocks(Vector3d heel,
+                                                Vector3d toe,
+                                                double wellbore_radius) {
+                       heel_ = heel;
+                        toe_ = toe;
             wellbore_radius_ = wellbore_radius;
+            vector<IntersectedCell> intersected_cells = cells_intersected();
 
-            std::vector<IntersectedCell> intersected_cells = cells_intersected();
-	    
             for (int i = 0; i < intersected_cells.size(); ++i) {
-                intersected_cells[i].set_well_index(compute_well_index(intersected_cells[i]));
+                intersected_cells[i].set_well_index(
```

Comment:
Ok, reverted to one line format given the new indenting makes the line just wide enough.